### ISSUE TITLE
no-trailing-spaces: Auto fix issue

### ISSUE THREAD

**Issue by gyandeeps** [eslint/eslint#3912]
**Eslint:** 1.5.1
**Parser:** Default

**Code:**
```js
"\n" +
"measAr.push(\"<dl></dl>\",  \n" +
"         \" </dt><dd class ='pta-res'>\");"
```

**Current Fix Output:**
```js
"\n" +
"measAr.push(\"<dl></dl>\" \n" +
"         \" </dt><dd class ='pta-res'>\");"
```

**Expected Fix Output:**
```js
"\n" +
"measAr.push(\"<dl></dl>\",\n" +
"         \" </dt><dd class ='pta-res'>\");"
```

**Issue:** On line 2, it got rid of the comma at the end. Which it shouldn't.

Confirmed: this is only happening when you have a empty line on top of the issue line.
